src/qp/factory.py

```diff
@@ -164,6 +164,20 @@ def read(self, filename):
             data = reader_convert(data)
         return Ensemble(ctor_func, data=data, ancil=ancil_table)
 
+    def data_length(self, filename):
+        """Get the size of data
+
+        Parameters
+        ----------
+        filename : `str`
+
+        Returns
+        -------
+        nrows : `int`
+        """
+        f, infp = io.readHdf5Group(filename, 'data')
```

Comment:
Is `infp` going to be used for something at a later date?

Reply:
Not it this function.   I'll set it to `f, _` instead.